about.html

```diff
@@ -0,0 +1,94 @@
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="style.css">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> 
+    <link rel="stylesheet" href="style2.css">
+<body>
+    <header>
+        <ul class="nav nav justify-content-end">
+            <li><a href="#" class="fa fa-facebook"></a></li>
+            <li><a href="#" class="fa fa-twitter"></a></li>
+            <li><a href="#">Home</a></li>
+            <li><a href="#">My page</a></li>
+            <li><a href="#">Logout</a></li>
+          </ul>
+    </header>
+    <div class="top">
+        <div class="top1">
+            <img alt="logo" src="Images/Logo.png">
+            <ul class="nav">
+                <li class="nav-item">
+                <a class="nav-link" href="index.html">Home</a>
+                </li>
+                <li class="nav-item">
+                <a class="nav-link" href="about.html">about</a>
+                </li>
+                <li class="nav-item">
+                <a class="nav-link" href="#">Join</a>
+                </li>
+                <li class="nav-item">
+                <a class="nav-link" href="#">Sponsors</a>
+                </li>
+                <li class="nav-item">
+                <a class="nav-link active" href="#">Contact</a>
+                </li>
+            </ul>
+        </div>
+    </div>
+    <div class="main">
+        <div class="overlay">
+            <div class="menu-overlay" onclick = "showMenu()">
+                <div class="menu">
+                    <div class="login">Login</div>
+                    <a class="menu-link" href="index.html">Home</a>
+                    <a class="menu-link" href="about.html">about</a>
+                    <a class="menu-link" href="#">Join</a>
+                    <a class="menu-link" href="#">Sponsors</a>
+                    <a class="menu-link" href="#">Contact</a>
+                </div>
+            </div>
+            <div class="navigation"  onclick = "showMenu()">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+            
+
+            <section class="intro" id="Intro">
+                <h1 class="main-title">THE WORLD  FOOTBALL GOVERNING BODY (FIFA)<br> HAS SETTLED ON RWANDA TO HOST THE 73rd FIFA CONGRESS</h1>
+                <div class="intro-message">
+                    <p>The Fédération internationale de football association is the international governing body of association football, beach soccer, and futsal. It was founded in 1904 to oversee international competition among the national associations of Belgium, Denmark, France, Germany, the Netherlands, Spain, Sweden and Switzerland</p>
+                </div>
+            </section>
```

Comment:
- [ ] This section contains a top section with a central column that includes a page title, a text block with the page description, and a line with contact information or any other prompt under it, but you don't have the line with contact information kindly add it 🙃 🙃 
- [ ] Please kindly decrease a little bit the size of your title and your paragraph as well, they are too big 🙃 🙃. Also, center your paragraph 👍
- [ ] Kindly use the same color as in the original design for the title 🙃 

> |     | Your design                     | Original design                     |     |
> | --- | ----------------------------------- | ------------------------------------ | --- |
> |     | <img width="1003" alt="Screen Shot 2023-02-15 at 23 52 43" src="https://user-images.githubusercontent.com/74501523/219159262-7ff86f4a-26d9-439b-b7a8-2dcd6521c7e0.png"> | <img width="879" alt="Screen Shot 2023-02-15 at 23 58 14" src="https://user-images.githubusercontent.com/74501523/219159514-d8c98e02-644a-4e3a-8b50-3121902194ea.png"> |